UI/window-basic-settings.cpp

```diff
@@ -3384,6 +3462,90 @@ void OBSBasicSettings::ReloadAudioSources()
 	LoadAudioSources();
 }
 
+void OBSBasicSettings::MultichannelToggled(bool state)
+{
+	if (state)
+	{ //display all channels
+		ui->channelSetup->addItem("2.1");
+		ui->channelSetup->addItem("4.0 Quad");
+		ui->channelSetup->addItem("4.1");
+		ui->channelSetup->addItem("5.1");
+		ui->channelSetup->addItem("7.1");
+		blog(LOG_INFO, "Multichannel ON, bitrates > 320 kbs unlocked");
+	//display all bitrates
+		PopulateAACBitrates({ ui->simpleOutputABitrate,
+				ui->advOutTrack1Bitrate, ui->advOutTrack2Bitrate,
+				ui->advOutTrack3Bitrate, ui->advOutTrack4Bitrate,
+				ui->advOutTrack5Bitrate, ui->advOutTrack6Bitrate });
+	}	
+	else
+	{ 
+		blog(LOG_INFO, "Multichannel OFF");
+	/* 
```

Comment:
Style:  This multi-line comment should be indented one more level.  After the first line, there needs to be one space preceding the asterisk.  Below is an example.
```
		/*
		 * reset channel and bitrates settings when Multichannel is switched OFF
		 * first reset channel menu and save new setting
		 */
```